bors.toml

```diff
@@ -1,6 +1,7 @@
 status = [ "continuous-integration/jenkins/branch" ]
 pr_status = [ "license/cla" ]
 required_approvals = 1
+block_labels = [ "blocked" ]
```

Comment:
This has traditionally been used as an issue management label rather than a PR management one (see [label description](https://github.com/Microsoft/openenclave/labels)) … we could repurpose it, in which case the description should be updated, or it may be better to introduce a specific label for this purpose that better reflects "there's a manual hold on checking this PR in".

Reply:
We'd already started repurposing it 😉 but that's why I added you. What about a "hold" label?

Reply:
Yeah, something like "on hold" is probably a clearer descriptor

Reply:
I made it "do not merge." It is direct and to the point 😄